client_cli/src/main.rs

```diff
@@ -147,7 +147,16 @@ fn main() -> Result<()> {
     let config = if let Some(config) = config_opt {
         config
     } else {
-        Configuration::from_str("config.json")?
+        let config_path = iroha::ConfigPath::new("config").with_preconfigured_extensions();
+        #[allow(clippy::expect_used)]
+        Configuration::from_str(
+            config_path
+                .first_existing_path()
+                .wrap_err("Configuration file does not exist")?
+                .as_ref()
+                .to_str()
+                .expect("Default config path is a valid UTF-8 string, qed."),
```

Comment:
```suggestion
                .expect("Default config path wasn't a  valid UTF-8 string."),
```

Remember this is an Error message. On that note, consider adding a `config_path` as an argument.

Reply:
I decided to use `to_string_lossy()` and remove this `expect()`. Also it will always valid, becuase it's the default `config.json` or `config.json5` path